frontend/public/robots.txt

```diff
@@ -1,3 +1,3 @@
 # https://www.robotstxt.org/robotstxt.html
-User-agent: *
-Disallow:
+User-agent: DatadogSynthetics
```

Comment:
this is intended ?

Reply:
Yes, We don't want to block all crawlers from crawling. So, I thought to disallow  DatadogSynthetics only.

Reply:
any specific reason to not stop all the crawlers ? and only DD crawler ?

Reply:
i am not much aware of the context here.. @YounixM for your eyes